### PR TITLE
Use same data type for pitch and angleturn

### PR DESCRIPTION
**Pull request by ceski-1**

### CODE REVIEW

File: src/d_ticcmd.h [hunk 1]
```diff
@@ -36,7 +36,7 @@ typedef struct ticcmd_s
     byte	chatchar;
     byte	buttons;
 
-    int pitch;
+    short pitch;
     short ticangleturn; // Local angle delta for composite input only.
```
Comment (rfomin) [inline src/d_ticcmd.h:40]: Would it be too much work if we removed both `pitch` and `ticangleturn` from the `ticcmd_t` structure? I added `pitch` a while ago, thinking that maybe we would send the mouselook command over the network, but these days I doubt that will ever happen.

I don't know yet why resizing `ticcmd_t` affects synchronization, but I think it's cleaner if we separate local data from network data.

Comment (ceski-1) [reply src/d_ticcmd.h:40]: I took a look and it's not something I want to mess with at the moment. There are too many headaches lurking in the code (free look toggle resets [everyone](https://github.com/fabiangreffrath/woof/blob/master/src/mn_setup.c#L2365)?).

Comment (rfomin) [reply src/d_ticcmd.h:40]: But `player->centering` is only local. Anyway, if there's a lot of work to do, we can postpone it for later.

**Review by rfomin — APPROVED** [review]

File: src/d_ticcmd.h (hunk 1 above)
Comment (ceski-1) [reply src/d_ticcmd.h:40]: Thanks, best for later I think.

**Pull request merged**